c2corg_ui/static/js/addassociation.js

```diff
@@ -83,11 +83,12 @@ app.AddAssociationController = function(appApi, appDocument) {
  */
 app.AddAssociationController.prototype.associate = function(doc) {
   var parentId, childId;
+  var type = this.parentDoctype;
```

Comment:
Please use ``parentType`` at least. Else it is difficult to understand the type of what document you are testing.